trick_source/web/README.md

```diff
@@ -3,4 +3,4 @@ Trick has the capability to create a custom built web server for your sim.
 
 You can find documentation here:
 
-[Web Server Documentation](https://nasa.github.io/trick/documentation/simulation_capabilities/web/)
+[Web Server Documentation](/docs/documentation/web/Adding_a_Web_Server_to_Your_Sim)
```

Comment:
```suggestion
[Web Server Documentation](https://nasa.github.io/trick/documentation/web/Adding_a_Web_Server_to_Your_Sim)
```

Need the full path to our hosted docs at nasa.github.io. Relative path will take you to documentation source instead of the stylized docs page.

Reply:
Ah, I see.